webserver/script/CostBenefitAnalysis/preprocessing_loadprofiles/split_file.py

```diff
@@ -0,0 +1,155 @@
+'''

```

Comment:
This code is originally from Robbie to preprocess the csv output from load forecast profile to then feed to the CBA tool. Some changes here from me.